### PR TITLE
Tm/runtime apprenticeships

### PR DESCRIPTION
**Pull request by basilesportif**
@jalehman @mikeosborne

### CODE REVIEW

**Comment by jalehman** [comment]
Why are apprenticeships being deleted @timlucmiptev ? It looks like there are contributors that have signed up to work on these ones specifically.

**Comment by basilesportif** [comment]
Ah, you're right, we shouldn't delete. I wanted to sort of mark them as
closed since I want people to do others.

Чт, 24 июня 2021 г. в 23:44, Josh Lehman ***@***.***>:

**Comment by basilesportif** [comment]
ok, undeleted the ones I don't want available and marked them `[Closed]`.

@mikeosborne Can you look at the `ra-*` runtime ones and see what to insert in the `TODO` parts?

File: content/grants/apprenticeships/ua-heads-up-poker.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 +++
-title = "Apprenticeship: Heads-up poker"
+title = "[Closed] Userspace Apprenticeship: Heads-Up Poker"
```
Comment (jalehman) [inline content/grants/apprenticeships/ua-heads-up-poker.md:2]: No need to write in `[Closed]` to the title—just set the `assignee` property to the `@p` of the person(s) that are working on it and it'll no longer be claimable.

**Review by jalehman — CHANGES_REQUESTED** [review]

**Comment by jalehman** [comment]
Also, this doesn't build due to errors in the page topmatter. Please run locally to ensure that everything builds (see README for instructions).

**Comment by basilesportif** [comment]
Should be good now -- @mikeosborne could you add work request links?

**Comment by jalehman** [comment]
I'll add the links in a follow-up PR.

**Pull request merged**